drivers/tapdisk-server.c

```diff
@@ -321,6 +323,9 @@ tapdisk_server_close_tlog(void)
 static void
 tapdisk_server_close(void)
 {
+	if (likely(server.tlog_reopen_evid >= 0))
+		tapdisk_server_unregister_event(server.tlog_reopen_evid);
+
```

Comment:
It might be a good idea to reset tlog_reopen_evid to -1 after unregistering the event